docs/scripts/co2_containment.rst

```diff
@@ -0,0 +1,48 @@
+
+CO2_CONTAINMENT
+==============
+
+Calculates the amount of CO\ :sub:`2` inside and outside a given perimeter, and separates the result per formation and phase (gas/dissolved). Output is a table on CSV format.
+
+Input

```

Comment:
Please consider using 

```rst
.. argparse::
   :module: subscript.co2_containment.co2_contaiment
   :func: get_parser
   :prog: co2_containment
```

Sphinx can create a nice listing of the inputs from the options supplied to argparse -- see the other scripts which do this. It helps to keep documentation up-to-date by minimizing the number of places it needs to be updated. But whichever may give the most information